cmd/soci/commands/create.go

```diff
@@ -30,7 +30,6 @@ import (
 
 const (
 	buildToolIdentifier = "AWS SOCI CLI"
```

Comment:
If you have invalid zTOC, wouldn't you want to track down the version of this CLI?

Reply:
Should we add the version to the identifier string?

Reply:
Since we have fbs representation for ztoc public, anyone can implement their own tooling to generate ztoc, so it will be hard to do so.

Reply:
Yes, we should add the version to the string.